ui/packages/shared/profile/src/ProfileView/index.tsx

```diff
@@ -263,17 +263,81 @@ export const ProfileView = ({
                   placeholderText="Add panel..."
                   primary
                   addView={true}
-                  disabled={!isSinglePanelView || dashboardItems.length < 1}
+                  disabled={isMultiPanelView || dashboardItems.length < 1}
                 />
               </div>
             </div>
 
             {isLoaderVisible ? (
               <>{loader}</>
             ) : (
-              <div ref={ref} className="flex space-x-4 justify-between w-full">
-                {dashboardItemsWithViewSelector}
-              </div>
+              <DragDropContext onDragEnd={onDragEnd}>
+                <div className="w-full" ref={ref}>
+                  <Droppable droppableId="droppable" direction="horizontal">
+                    {provided => (
+                      <div
+                        ref={provided.innerRef}
+                        className="flex space-x-4 justify-between w-full"
+                        {...provided.droppableProps}
+                      >
+                        {dashboardItems.map((dashboardItem, index) => {
+                          return (
+                            <Draggable
+                              key={dashboardItem}
+                              draggableId={dashboardItem}
+                              index={index}
+                              isDragDisabled={!isMultiPanelView}
+                            >
+                              {(provided, snapshot: {isDragging: boolean}) => (
+                                <div
+                                  ref={provided.innerRef}
+                                  {...provided.draggableProps}
+                                  key={dashboardItem}
+                                  className={cx(
+                                    'border dark:bg-gray-700 rounded border-gray-300 dark:border-gray-500 p-3',
+                                    isMultiPanelView ? 'w-1/2' : 'w-full',
+                                    snapshot.isDragging ? 'bg-gray-200' : 'bg-white'
+                                  )}
+                                >
+                                  <div className="w-full flex justify-end pb-2">
+                                    <div className="w-full flex justify-between">
+                                      <div
+                                        className={cx(isMultiPanelView ? 'visible' : 'invisible')}
+                                        {...provided.dragHandleProps}
+                                      >
+                                        <DragIndicator />
```

Comment:
Shall we replace this icon with the [dots drag handle icon](https://icones.netlify.app/collection/all?s=material-symbols:drag-indicator)? `<Icon icon="material-symbols:drag-indicator" />`



I feel that icon is more universal, and this one looks a bit confusing, like the hamburger menu.

Reply:
> 

Yesss! That's a great point. I think the reset button is extraneous in this case - will remove it